lib/utils.pm

```diff
@@ -957,7 +957,13 @@ sub ensure_serialdev_permissions {
     # ownership has effect immediately, group change is for effect after
     # reboot an alternative https://superuser.com/a/609141/327890 would need
     # handling of optional sudo password prompt within the exec
-    assert_script_run "chown $testapi::username /dev/$testapi::serialdev && gpasswd -a $testapi::username \$(stat -c %G /dev/$testapi::serialdev)";
+    if (is_sle && check_var('VERSION', '11-SP4')) {
+        # Backwards support for SLES11, gpasswd or stat commmand is not supprt on SLES11
```

Comment:
Please make it clear why separate action is required for 11-SP4, the ambiguous and uncertain description like "gpasswd or stat commmand is not supprt on SLES11" is NOT accepted.